src/common/utils/auth_tests.rs

```diff
@@ -0,0 +1,1202 @@
+                method: format!("{PUT_METHOD}"),
+                o2_type: format!("settings:{ORG_ID}"),
+                org_id: format!("{ORG_ID}"),
+                bypass_check: false,
+                parent_id: format!("default"),
+            },
+        )
+        .await
+    }
+
+    #[tokio::test]
+    async fn get_organization_settings() {
+        test_auth(
+            Method::GET,
+            format!("api/{ORG_ID}/settings"),
+            AuthExtractor {
+                auth: format!("{AUTH_HEADER_VAL}"),
+                method: format!("{GET_METHOD}"),
+                o2_type: format!("settings:{ORG_ID}"),
+                org_id: format!("{ORG_ID}"),
+                bypass_check: false,
+                parent_id: format!("default"),
+            },
+        )
+        .await
+    }
+
+    #[tokio::test]
+    async fn upload_logo() {
+        test_auth(
+            Method::POST,
+            format!("api/{ORG_ID}/settings/logo"),
+            AuthExtractor {
+                auth: format!("{AUTH_HEADER_VAL}"),
+                method: format!("{PUT_METHOD}"),
+                o2_type: format!("settings:{ORG_ID}"),
+                org_id: format!("{ORG_ID}"),
+                bypass_check: false,
+                parent_id: format!("default"),
+            },
+        )
+        .await
+    }
+
+    #[tokio::test]
+    async fn delete_logo() {
+        test_auth(
+            Method::DELETE,
+            format!("api/{ORG_ID}/settings/logo"),
+            AuthExtractor {
+                auth: format!("{AUTH_HEADER_VAL}"),
+                method: format!("{PUT_METHOD}"),
+                o2_type: format!("settings:{ORG_ID}"),
+                org_id: format!("{ORG_ID}"),
+                bypass_check: false,
+                parent_id: format!("default"),
+            },
+        )
+        .await
+    }
+
+    #[tokio::test]
+    async fn set_logo_test() {
+        test_auth(
+            Method::POST,
+            format!("api/{ORG_ID}/settings/logo/text"),
+            AuthExtractor {
+                auth: format!("{AUTH_HEADER_VAL}"),
+                method: format!("{PUT_METHOD}"),
+                o2_type: format!("settings:{ORG_ID}"),
+                org_id: format!("{ORG_ID}"),
+                bypass_check: false,
+                parent_id: format!("default"),
+            },
+        )
+        .await
+    }
+
+    #[tokio::test]
+    async fn delete_logo_text() {
+        test_auth(
+            Method::DELETE,
+            format!("api/{ORG_ID}/settings/logo/text"),
+            AuthExtractor {
+                auth: format!("{AUTH_HEADER_VAL}"),
+                method: format!("{PUT_METHOD}"),
+                o2_type: format!("settings:{ORG_ID}"),
+                org_id: format!("{ORG_ID}"),
+                bypass_check: false,
+                parent_id: format!("default"),
+            },
+        )
+        .await
+    }
+
+    // routes defined in handler::http::request::organization::es
```

Comment:
Ok, so currently looks like we ignore the es endpoints in the openfga checks, I will check if we need to change anything here.

Reply:
Good to know. Thanks! I'll wait to hear back from you before moving forward with this PR.